docs/source/intro-clx-loda-anomaly-detection.ipynb

```diff
@@ -0,0 +1,171 @@
+{
```

Comment:
<p>define what these variables (n_bins, n_random_cuts) mean</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>n_bins: Number of bins in each histogram. </p><p>n_random_cuts: Number of random cut projections. </p>

Reply:
<p>updated with the variable definition</p>